### DISCUSSION TITLE
Is there a way to check whether a plugin is installed?

### DISCUSSION THREAD

**Discussion by glyh** [wbthomason/packer.nvim#754] [Q&A] [answered]
I have the following code in my mapping confs:
```lua
nvim.cmd([[
  try
    call arpeggio#map('i', '', 0, 'jk', '<Esc>')
  catch
  endtry
]])
```
which depends on some specific plugin. I'd like to know if there are other ways to detect whether some plugin is installed? Thanks a lot!

**Comment by hinell** [comment, marked as answer]
Thumbing up this question. Currently we can only check whether [it's loaded or not](https://github.com/wbthomason/packer.nvim#checking-plugin-statuses). Any tips?

**Reply by glyh** [reply]
I think this solves my problem, thanks.

**Reply by hinell** [reply]
There is a way to checkout whether some plugin is loadable or not btw:
```lua
local ok, mod = pcall(require, "foo/bar")
if ok then 
...
end
```
